RELEASE_NOTES.md

```diff
@@ -1,7 +1,8 @@
 # Upcoming
 
-- fix: Miscompilation due to incorrect parenthesization in C# output for casts. (#1908)
+- fix: Miscompilation due to incorrect parenthesization in C# output for casts. (https://github.com/dafny-lang/dafny/pull/1908)
```

Comment:
Why do you add the full link?

Reply:
So that it's browsable when reading the README.md from a text editor, and in Github markdown it still displays nicely "#1908"